### PR TITLE
Add Hebrew translations

### PR DESCRIPTION
**Pull request by ronbentata**
**Summary**

Added an Hebrew translation + ReadMe

**Translation checklist**

- [x] I have ran the application to make sure my code runs
- [x] I have used the correct [ISO 639-1 code](https://www.alchemysoftware.com/livedocs/ezscript/Topics/Catalyst/Language.htm) for my translations (e.g. `en`)
- [x] I have made sure to leave a `,` at the end of each translation added
- [x] I have added the language to `supportedLanguages` in alphabetical order
- [x] I have added the language to the table in README.md in alphabetical order
- [x] (_If you translated the README_) I have translated the README and linked it in the main README.md

### CODE REVIEW

File: README-Hebrew.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+עיין ב-[CONTRIBUTING.md](./CONTRIBUTING.md) כדי לראות הנחיות כיצד לתרום. הקובץ מתאר מה לעשות אם יש לך תכונה חדשה, שינוי, עדכון תרגום או שמצאת בעיה עם auto-py-to-exe.
+
+## בעיות בשימוש בכלי
+
+אם אתה נתקל בבעיות עם קובץ ההפעלה הארוז או בשימוש בכלי זה באופן כללי, אני ממליץ לך לקרוא את [הפוסט בבלוג שלי על בעיות נפוצות בעת שימוש ב-auto-py-to-exe](https://nitratine.net/blog/post /issues-when-using-auto-py-to-exe/?utm_source=auto_py_to_exe&utm_medium=readme_link&utm_campaign=auto_py_to_exe_help). פוסט זה מכסה דברים שכדאי לדעת על אריזת סקריפטים של Python ותיקונים לדברים שבדרך כלל משתבשים.
+
+אם אתה סבור שמצאת בעיה בכלי זה, עקוב אחר הסעיף ["דיווח על בעיה" ב-CONTRIBUTING.md](./CONTRIBUTING.md#reporting-an-issue).
+
+## תרגומים
```
Comment (brentvollebregt) [inline README-Hebrew.md:135]: The translation table can be left out

File: auto_py_to_exe/web/js/i18n.js [hunk 2]
```diff
@@ -357,7 +369,8 @@ const translationMap = {
         ko: '출력 디렉토리',
         bg: 'Изходяща директория',
       },
-      increaseRecursionLimit: {
+      increaseRecursionLimhe: '',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:372]: I see you checked "I have ran the application to make sure my code runs" - you may want to run it again as it appears this stops the application from loading.

File: auto_py_to_exe/web/js/i18n.js [hunk 3]
```diff
@@ -1977,6 +2044,7 @@ const translationMap = {
         th: 'การดำเนินการนี้จะเขียนทับเอาต์พุตก่อนหน้าในโฟลเดอร์เอาต์พุต\nทำต่อหรือไม่',
         fr: 'Cette action écrasera une sortie précédente dans le dossier de sortie.\nContinuer ?',
         hi: 'यह कार्रवाई आउटपुट फ़ोल्डर में पिछले आउटपुट को अधिलेखित कर देगी। \n क्या आप जारी रखना चाहते हैं?',
+        he: '',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2047]: If there is no translation, it can be left out

File: auto_py_to_exe/web/js/i18n.js [hunk 4]
```diff
@@ -2136,6 +2204,10 @@ const supportedLanguages = [
     name: 'Greek (Ελληνικά)',
     code: 'el',
   },
+  {
+    name: 'Hebrew (עברית)',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:2208]: You checked "I have added the language to supportedLanguages in alphabetical order" - maybe you forgot to commit the change? It is not in this PR.

File: auto_py_to_exe/web/js/i18n.js [hunk 5]
```diff
@@ -1173,6 +1213,9 @@ const translationMap = {
           'Onefile मोड के साथ अतिरिक्त फ़ाइलों का उपयोग करते समय सावधान रहें।;\n' +
           `<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>read this</a>\n` +
           'और अपने कोड को PyInstaller के साथ काम करने के लिए अपडेट करें।',
+        he: 'היזהר בעת שימוש בקבצים נוספים במצב קובץ-יחיד;\n' +
+        `<a ${onFileModeAdditionalFilesHelpAnchorTagContents}>קרא את זה</a>\n` +
+        'ועדכן את הקוד שלך לעבודה עם PyInstaller.',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:1218]: Please format this code as specified in the [contribution guidelines](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/CONTRIBUTING.md#style-guide) - we use prettier

**Review by brentvollebregt — CHANGES_REQUESTED** [review]

File: auto_py_to_exe/web/js/i18n.js (hunk 2 above)
Comment (ronbentata) [reply auto_py_to_exe/web/js/i18n.js:372]: I have. must be something of last change I haven't check specifically. 
I've resolved now

File: auto_py_to_exe/web/js/i18n.js (hunk 3 above)
Comment (ronbentata) [reply auto_py_to_exe/web/js/i18n.js:2047]: updated

File: auto_py_to_exe/web/js/i18n.js (hunk 4 above)
Comment (ronbentata) [reply auto_py_to_exe/web/js/i18n.js:2208]: yes. added it, between Hindi and Greek

File: README-Hebrew.md (hunk 1 above)
Comment (ronbentata) [reply README-Hebrew.md:135]: not clear on your meaning. I have translated only the columns' captions, not the table contents. should I change/revert something?

File: auto_py_to_exe/web/js/i18n.js (hunk 5 above)
Comment (ronbentata) [reply auto_py_to_exe/web/js/i18n.js]: I've adjusted the format

**Comment by ronbentata** [comment]
please update me regarding issue 1 and 4 (you mentioned and I responded). 
if it's ok with you, I'll push the latest updates

File: README-Hebrew.md (hunk 1 above)
Comment (brentvollebregt) [reply README-Hebrew.md:135]: Check out the other translated README files - notice they do not have the "Translations" section. This section is only in the README.md.

File: auto_py_to_exe/web/js/i18n.js (hunk 4 above)
Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js:2207]: My bad, I was meant to reference "I have added the language to the table in README.md in alphabetical order". The changes need to also be in README.md - [in this block](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md?plain=1#L130).

File: auto_py_to_exe/web/js/i18n.js (hunk 5 above)
Comment (brentvollebregt) [reply auto_py_to_exe/web/js/i18n.js]: I ran the formatter locally with your changes and there were still formatting differences. Might pay to run it again and commit all the differences - there are more than just this one.

**Comment by ronbentata** [comment]
pushed new commit to resolve latest mentioned issues

File: auto_py_to_exe/web/js/i18n.js [hunk 6]
```diff
@@ -1031,7 +1066,8 @@ const translationMap = {
         ko: '결과물이 저장될 디렉토리. 디렉토리가 존재하지 않으면 자동 생성됩니다.',
         bg: 'Изходната папка ще бъде създадена, ако не съществува.',
       },
-      increaseRecursionLimit: {
+      increaseRecursionLimhe: '',
+        it: {
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:1070]: Are you sure this runs? This is the same error that caused it not to run previously.

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
The formatting check failed again, here is the output from the action run:

```
Checking formatting...
[warn] auto_py_to_exe/web/js/i18n.js
[warn] README-Hebrew.md
[warn] README.md
[warn] Code style issues found in 3 files. Forgot to run Prettier?
Problem running prettier with --check .
```

File: auto_py_to_exe/web/js/i18n.js (hunk 6 above)
Comment (ronbentata) [reply auto_py_to_exe/web/js/i18n.js]: no. this is old. I pushed another commit with fixed code. 
I have now pushed (again) fix for this. 
sorry for the repo confusion and code ping-pong.

**Comment by brentvollebregt** [comment]
Hey @ronbentata, not sure if you saw my couple of comments about formatting - this still needs to be sorted before merge.

**Comment by ronbentata** [comment]
run prettier again to fix format on specific files

File: auto_py_to_exe/web/js/i18n.js [hunk 7]
```diff
@@ -312,6 +322,7 @@ const translationMap = {
         tr: `${applicationName} ye Özel Seçenekler`,
         th: `${applicationName} ตัวเลือกเฉพาะ`,
         fr: 'Options spécifiques',
+        he: 'אפשרויות ייחודיות של ${applicationName}',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:325]: This is not using the correct quotes - `${applicationName}` will not be evaluated

**Review by brentvollebregt — APPROVED** [review]
Cheers, one more thing I spotted when I ran the application myself. After this, it will be good to merge 👍

File: auto_py_to_exe/web/js/i18n.js (hunk 7 above)
Comment (ronbentata) [reply auto_py_to_exe/web/js/i18n.js]: - updated

**Review by brentvollebregt — APPROVED** [review]

**Comment by ronbentata** [comment]
Thank you!
great to contribute



*Ron Bentata*
LinkedIn:     https://www.linkedin.com/in/ronbentata
Website:      http://www.ronbentata.com/
please consider the environment and don't print this email

**Pull request merged**